features/pageobjects/login.page.js

```diff
@@ -0,0 +1,28 @@
+const Page = require('./page');
+ 
+class LoginPage extends Page {
+    get inputUsername () {
+        return $('[name="email"]');
+    }
+
+    get inputPassword () {
+        return $('[name="password"]');
+    }
+
+    get btnSubmit () {
+        return $('button[type="submit"]');
```

Comment:
Did you consider moving button[type="submit"] in some common page?
As this element can be duplicated for many pages